keras_cv/layers/preprocessing/solarization.py

```diff
@@ -67,6 +88,10 @@ def _solarize_above_threshold(self, images):
         return tf.where(images < self.threshold, images, self._solarize(images))
```

Comment:
Maybe rename this method to _solarize().

Reply:
Done